openshift/github-mirror.yaml

```diff
@@ -13,6 +13,10 @@ objects:
     selector:
       matchLabels:
         app: github-mirror
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: ${SERVICE_ACCOUNT}
```

Comment:
I think you are also going to need a role that allows this service account to read the `EC_SECRET_NAME` secret